OracleWebLogic/samples/12213-webtier-apache/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# Copyright (c) 2016-2018 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+docker build -t 12213-apache .
```

Comment:
This is not acceptable as a script. Please review the `buildDockerImage.sh` scripts used in the repo to provide more flexibility when building this image.